src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,50 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortSubmission
```

Comment:
In the sentence "This doesn't make sense as a logical unit." what do you refer to by "this"?

Maybe it should be named "AbortButton" instead of "AbortSubmission"?  I had thought to name it "AbortButton", but said  "it's a div, not a button", so then thought "AbortDiv" should be a name...but that might be confusing so then I thought "AbortSubmission" which is maybe confusing too.

By "status display" do you refer to the div whose backgroundcolor calls the "color-for-submission"?